AdventOfCode2021/day1_1/day1_1.py

```diff
@@ -0,0 +1,24 @@
+#!python3
+
+from typing import List
+
+def file_read(file_name) -> List[int]:
+    file_data_list: List[int] = []
```

Comment:
When you have a variable that accumulates the result of the function it's often useful to name it `result`.